shell/components/CruResource.vue

```diff
@@ -257,10 +266,30 @@ export default {
       this.$emit('select-type', id);
     },
 
+    clickSave(e) {
+      this.createNamespaceIfNeeded();
+      this.$emit('finish', e);
+    },
+
     save() {
       this.$refs.save.clicked();
     },
 
+    async createNamespaceIfNeeded() {
```

Comment:
This async call wouldn't work for epinio - would need a custom callback passed in